Building with Graphics none and Platform none causes include error

## **Description : Running the emake command with graphics none and platform none gives an include error.**

#### basically when i run " ./emake -p None -g None -o /tmp/test " in the enigma-dev directory i get this error 

```
./emake -p None -g None -o /tmp/test
LibEGM parsing log at: C:\msys64\tmp\/enigma_libegm.log
ENIGMA compiler log at: C:\msys64\tmp\/enigma_compiler.log
Linking up to IDE
Implementing JDI basics
UnimplementedUnimplemented

Choosing language: C++
 > Normal
 > Stacked
 > Specialized
 > TriggerOnce
 > TriggerAll
 > Inline
Creating parse context
Warning: No game file specified. Building an empty game.
Flattening tree.
- Not transferring unknown
Transfer complete.

Initializing dialog boxes
Initialized.
Building for mode (1)
Loading shared locals from extensions list
The IDE didn't tell ENIGMA what extensions were selected before requesting a build.Extensions I have loaded:
- Universal_System/Extensions/Alarms
- Universal_System/Extensions/DataStructures
- Universal_System/Extensions/DateTime
- Universal_System/Extensions/GTest
- Universal_System/Extensions/MotionPlanning
- Universal_System/Extensions/Paths
- Universal_System/Extensions/Timelines
- Universal_System/Extensions/libpng
Extensions I should have loaded:
...Continuing anyway...
Copying resources:
Copying sprite names [0]
Copying sound names [0]
Copying background names [0]
Copying path names [0]
Copying script names [0]
Copying shader names [0]
Copying font names [0]
Copying timeline names [0]
Copying object names [0]
Copying room names [0]
Copying constant names [0]
SYNTAX CHECKING AND PRIMARY PARSING:
0 Scripts:
"Linking" scripts
`Linking' 0 scripts and 0 timelines in 0 passes...
Completing script "Link"
Done.
Completing timeline "Link"
Done.
0 Objects:
Creating room creation code scope and parsing
"Linking" scripts into the objects...
"Linking" timelines into the objects...
"Link" complete.
Tabulating maximum argument passes to each script
Finished
Writing executable information and resources.
Writing modes and settings
Writing object switch
Writing resource names and maxima
Writing timeline control information
Linking globals and ambiguous variables
Running Secondary Parse Passes
Writing events
Writing object data
Writing local accessors
Writing font data
Writing room data
Writing shader data
Running make from `make'
Full command line: make -C "C:/enigma/enigma-dev/" Game WORKDIR="C:/Users/abhis/AppData/Local/ENIGMA/" CODEGEN="C:/Users/abhis/AppData/Local/ENIGMA/" GMODE="Debug" GRAPHICS="None" AUDIO="OpenAL" COLLISION="BBox" WIDGETS="None" NETWORKING="None" PLATFORM="None" TARGET-PLATFORM="Windows" CXXFLAGS="-fdiagnostics-color=always -I../Additional/i686-w64-mingw32/include" LDFLAGS="-L../Additional/i686-w64-mingw32/lib -static-libgcc -static-libstdc++ -static" MAKE="make" COMPILEPATH="Windows/Windows/Mingw_GCC_G++" EXTENSIONS=" Universal_System/Extensions/Alarms Universal_System/Extensions/Paths Universal_System/Extensions/libpng Universal_System/Extensions/DataStructures Universal_System/Extensions/Timelines Universal_System/Extensions/MotionPlanning Universal_System/Extensions/DateTime Universal_System/Extensions/GTest" OUTPUTNAME="C:/msys64/tmp/test.exe"
make: Entering directory '/c/enigma/enigma-dev'
make[1]: Entering directory '/c/enigma/enigma-dev/ENIGMAsystem/SHELL'

################ Compiler Info ################
g++.exe (Rev2, Built by MSYS2 project) 10.3.0
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

CXXFLAGS="-fdiagnostics-color=always -I../Additional/i686-w64-mingw32/include -std=c++17 -Wall -g -DDEBUG_MODE -IC:/msys64/mingw64/include -IC:/msys64/mingw64/include/AL -DPATH_EXT_SET -I../../shared/libpng-util -IPlatforms/None/Info -IGraphics_Systems/None/Info -IAudio_Systems/OpenAL/Info -ICollision_Systems/BBox/Info -IWidget_Systems/None/Info -INetworking_Systems/None/Info -IUniversal_System/Info -I. -IC:/Users/abhis/AppData/Local/ENIGMA/ -I../../shared "
CFLAGS="-Wall -g -DDEBUG_MODE -IPlatforms/None/Info -IGraphics_Systems/None/Info -IAudio_Systems/OpenAL/Info -ICollision_Systems/BBox/Info -IWidget_Systems/None/Info -INetworking_Systems/None/Info -IUniversal_System/Info -I. -IC:/Users/abhis/AppData/Local/ENIGMA/ -I../../shared"
LDFLAGS="-L../Additional/i686-w64-mingw32/lib -static-libgcc -static-libstdc++ -static"
LDLIBS=""
################################################

[g++] SHELLmain.cpp
[g++] Platforms/None/fillin.cpp
Platforms/None/fillin.cpp:31:10: fatal error: sys/resource.h: No such file or directory
   31 | #include <sys/resource.h>
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:166: C:/Users/abhis/AppData/Local/ENIGMA/.eobjs/Windows/Windows/Mingw_GCC_G++/Debug/Platforms/None/fillin.o] Error 1
make[1]: Leaving directory '/c/enigma/enigma-dev/ENIGMAsystem/SHELL'
make: *** [Makefile:25: Game] Error 2
make: Leaving directory '/c/enigma/enigma-dev'


********* EXECUTE:
make -C "C:/enigma/enigma-dev/" Game WORKDIR="C:/Users/abhis/AppData/Local/ENIGMA/" CODEGEN="C:/Users/abhis/AppData/Local/ENIGMA/" GMODE="Debug" GRAPHICS="None" AUDIO="OpenAL" COLLISION="BBox" WIDGETS="None" NETWORKING="None" PLATFORM="None" TARGET-PLATFORM="Windows" CXXFLAGS="-fdiagnostics-color=always -I../Additional/i686-w64-mingw32/include" LDFLAGS="-L../Additional/i686-w64-mingw32/lib -static-libgcc -static-libstdc++ -static" MAKE="make" COMPILEPATH="Windows/Windows/Mingw_GCC_G++" EXTENSIONS=" Universal_System/Extensions/Alarms Universal_System/Extensions/Paths Universal_System/Extensions/libpng Universal_System/Extensions/DataStructures Universal_System/Extensions/Timelines Universal_System/Extensions/MotionPlanning Universal_System/Extensions/DateTime Universal_System/Extensions/GTest" OUTPUTNAME="C:/msys64/tmp/test.exe"
```

--- a/ENIGMAsystem/SHELL/Platforms/None/fillin.cpp
+++ b/ENIGMAsystem/SHELL/Platforms/None/fillin.cpp
@@ -25,16 +25,8 @@
 #include "Universal_System/roomsystem.h"
 #include "Universal_System/var4.h"
 
-#include <stdio.h>
-#include <stdlib.h>  //malloc
 #include <stdlib.h>  //getenv and system
-#include <sys/resource.h>
-#include <sys/stat.h>
-#include <unistd.h>
-#include <cstdlib>
-#include <map>
 #include <string>
-#include <time.h>
 
 namespace enigma {
   bool initGameWindow() { return true; }

